internals/secrethub/run_test.go

```diff
@@ -531,6 +536,7 @@ func TestRunCommand_Run(t *testing.T) {
 		},
 		"invalid template var: illegal character": {
```

Comment:
The variable reader is only used when there is an env file provided, thus the validation and the parsing of template variables is only done if `RunCommand.envFile != ""`, as opposed to the current release of the cli, which errors for invalid template variables even if `envFile == ""`. Do you think this is a problem, @SimonBarendse ?

Reply:
Discussed this in person. Not a problem.